scss/grid.scss

```diff
@@ -0,0 +1,95 @@
+.flex-body { 
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  grid-template-areas: 
+    "header"
+    "main"
+    "footer";
```

Comment:
If you use `grid-template-areas` you dont need template-columns nor template-rows.
Areas like `"header main footer"` will be 3 columns and 1 row.
Areas like  `"header" "main" "footer` will be 3 rows and 1 column.